packages/stream-model-instance-handler/src/__tests__/model-instance-document-handler.test.ts

```diff
@@ -326,10 +423,19 @@ describe('ModelInstanceDocumentHandler', () => {
   })
 
   it('makes signed commit correctly', async () => {
+    const content1 = {
```

Comment:
can stick with the CONTENTX constants defined above, no need to introduce new content vars in the test

Reply:
In fact, I think all changes to this test case can be reverted?